### PR TITLE
Kicking this thing off

### PR DESCRIPTION
**Pull request by kasima**
Starting with Introduction and Merkle Trees. Code and graphics to follow.

@davidknott, @kfichter – Would :heart: any feedback. Especially things I'm missing in the outline (basic concepts, faults) or if I'm misunderstanding or mischaracterizing anything.

### CODE REVIEW

File: 0-preface.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Preface
+
+## Who am I?
+
+My interest in Bitcoin started while I was working at GitHub in 2013. As part of the billing team (aptly named "gitcoin"), one of the projects we worked on was to make it easier for people around the world to subscribe. That opened the door to understanding just how hard it is for large parts of the world to participate in a credit card based system – something we take for granted in the West. I got excited about Bitcoin for its potential as a global payment network.
```
Comment (DavidKnott) [inline 0-preface.md]: It could be fun if you reference [gitcoin](https://gitcoin.co/) 😄

File: 1-introduction.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+Plasma is currently under heavy research and development by OmiseGO and multiple teams around the world. The white paper specifies many concepts that are yet to be explored for implementation. In the [Specifications](specifications) section, you can see more concrete specifications of Plasma, as well as implementations of those specs.
+
+
+## Structure
+
+In the most general case, Plasma describes the interaction of a tree of blockchains.
+
+[ XXX - add graphic ]
+
+Going from the leaf chains to the root of the tree, each chain periodically submits a summary update of its state to the parent chain. Eventually, the update of all the updates get submitted to a root chain (e.g. Ethereum). Conceptually, this is much like MapReduce. We map the update procedure through the tree and repeatedly reduce the results.
```
Comment (DavidKnott) [inline 1-introduction.md]: The white paper references MapReduce, but I'm having a little trouble seeing how Map fits here.  At least right now a tree of Plasma chains collects transactions at its tips and then Reduces them up into its parents and eventually the root chain.  Would you mind explaining this more?

File: 1-introduction.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+At OmiseGO and for the rest of these posts, we'll be using Ethereum as our root chain. That means we'll be using Solidity as our smart contract language in the examples.
+
+
+## Plasma Chain
+
+All the blockchains that are children of the root are called _Plasma chains_. Each Plasma chain will operate as a child and potentially as a parent too, depending on whether it's a an internal node or leaf in the tree.
+
+### As a child chain
+
+The Plasma chain acts as a child by periodically reporting an update back to its parent chain (which can also the be the root chain). This magical update up to the parent chain takes the form of the _Merkle root of a Merkle tree of transactions_ from the most recent block of the child chain.
```
Comment (DavidKnott) [inline 1-introduction.md]: A merkle root of a merkle tree of transactions is fairly spec/implementation specific, though it's what we're currently focused on.

File: basic_concepts/merkle_tree.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+[Merkle Trees](https://en.wikipedia.org/wiki/Merkle_tree), or Hash Trees, are trees (usually binary) of hashes starting with hashing the data at the leaf nodes. The hashes are hashed together up the tree until we reach a single root hash, also called the Merkle Root.
+
+[ XXX - graphic of full merkle tree – used below as well ]
+
+[ XXX - code to build a bninary merkle tree ]
+
+
+## Properties
+
+Merkle trees allow you to verify data at a leaf node efficiently.
```
Comment (DavidKnott) [inline basic_concepts/merkle_tree.md]: Add mention something about verifying the data through proofs of inclusion.

**Review by DavidKnott — COMMENTED** [review]
Left a few comments, though it generally looks good!

File: 1-introduction.md (hunk 3 above)
Comment (kasima) [reply 1-introduction.md]: Which part? Would you feel better about saying `state transitions`?

File: 1-introduction.md (hunk 2 above)
Comment (kasima) [reply 1-introduction.md]: I'm looking at it as an implicit Map, as enforced by the protocol. Am I understanding this wrong?

Comment (DavidKnott) [reply 1-introduction.md]: Can you explain more about how it's being mapped implicitly?

File: 1-introduction.md (hunk 3 above)
Comment (DavidKnott) [reply 1-introduction.md]: I'd say something along the lines of: "In the case of Plasma MVP, this magical update..."

**Pull request merged**